js_machine_front/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
```

Comment:
please add baseUrl to compileOptions './src'

Reply:
Fix